### DISCUSSION TITLE
Discord Server

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#3023] [Announcements]
Do you think there's a need for a Helper-Scripts Discord Server? Please share your thoughts below and suggest some channel names.

**Edit:** After thorough consideration, I have decided not to proceed with supporting a Helper-Scripts Discord server. The primary reason for this decision is my current lack of time to dedicate to effectively monitoring and managing the project, and having someone else run this for me would not provide the same interactions as we currently have here. I also feel that support would be too spread out.

**Comment by di5cord20** [comment]
I've gotten use to coming to the Github discussion daily to follow the latest and greatest.  I also really like the new site which shows me quickly any new scripts created (or I go to the changelog).  That said, for every other project I follow, I do so in Discord.  So moving tteck / helper-scripts to discord would be a welcomed change. 

I like how Discord provides the ability for dedicated channels (i.e., support/help, announcements, polls, resources such as guides or tutorials, etc.).  I see this community growing to help one another with their homelab questions and projects and Discord does that nicely.

Just my 2 cents...

**Comment by DSDV** [comment]
i would rather stay here and in discussions than a gated community which is not even searchable from the outside and based on a company which has a lot of (to me) strange policy's. 
especially if it has to do with keeping up with foss projects :/ 

i see a valid point on a "backup" platform tho in case sth happens to github or this repository.

**Comment by oOStroudyOo** [comment]
In my opinion, I think Discord can sometimes become too much "noise" if not organised well.

I'm happy staying here, but also open to joining Discord.
In my opinion the only thing I'd change about Github is how ideas are submitted. That ideas thread is over 1000 comments long, full of repeated ideas and lots of comments that don't include ideas.
Maybe migrating that into an Ideas forum on Discord would be a good move.

**Reply by DSDV** [reply]
i agree with the problems of the ideas thread : / 
maybe a kanban board or sth like that with voting would be a good alternative (i have seen some nice solutions from game devs [have to look them up but basically [like this concept](https://featureupvote.com/feature-voting-tool/) ]) 
but i don't see how a discord chat would be better. can you maybe elaborate?

**Comment by Mellowlynx** [comment]
I completely agree with both di5cord20 and oOStroudyOo.

I almost check in here daily, and think that a Discord server will help build a community, and share ideas and help one another.
But the "noise" is a big thing and also your support/interaction is something to think about. Do you have time to be there also?

I don't think there is a "need" for it now, but think it might be fun and if you deside to make one, I will join, and we will see how it go's.

**Reply by tteck** [reply]
Good point. I have edited the OP.

**Comment by lockenkop** [comment]
Discord not being indexed by search engines (obviously not easily possible) is the biggest dealbreaker for me. All the questions and answers get lost in a discord channel. If Stackoverflow was a discord server from the start, we would not have all those great knowledge at our hands. Discords search function is nothing to brag about either.
